evap/results/tools.py

```diff
@@ -165,7 +165,7 @@ def _calculate_results_impl(course):
             elif question.is_heading_question:
                 results.append(HeadingResult(question=question))
 
-        section_warning = questionnaire_max_answers[(questionnaire, contribution)] < questionnaire_warning_thresholds[questionnaire]
+        section_warning = questionnaire_max_answers[(questionnaire, contribution)] > 0 and questionnaire_max_answers[(questionnaire, contribution)] < questionnaire_warning_thresholds[questionnaire]
```

Comment:
why is this necessary?

if it stays, you can actually do `0 < questionnaire_max_answers[(questionnaire, contribution)] < questionnaire_warning_thresholds[questionnaire]`.

Reply:
otherwise the warning "only a few ..." would be shown for sections where no answers exist at all. changed the condition.